.github/workflows/build.yml

```diff
@@ -69,11 +71,76 @@ jobs:
           version: tags/20240511.0
           file: firebase-windows-amd64.zip
 
-      - run: Expand-Archive -Path firebase-windows-amd64.zip -DestinationPath third_party; Rename-Item -Path third_party/firebase -NewName firebase-development
+      - run: |
+          Expand-Archive -Path firebase-windows-amd64.zip -DestinationPath third_party
+          Rename-Item -Path third_party/firebase -NewName firebase-development
         shell: powershell
 
       - name: Configure
-        run: cmake -B out -D CMAKE_BUILD_TYPE=Release -G Ninja -S ${{ github.workspace }} -D CMAKE_Swift_FLAGS="${{ matrix.options }}" -D SWIFT_FIREBASE_BUILD_EXAMPLES=NO
+        run: |
+          cmake -B out `
+                -D CMAKE_BUILD_TYPE=Release `
+                -G Ninja -S ${{ github.workspace }} `
+                -D CMAKE_Swift_FLAGS="${{ matrix.options }}" `
+                -D SWIFT_FIREBASE_BUILD_EXAMPLES=NO
+
+      - name: Build
+        run: cmake --build out
+
+  android:
+    runs-on: windows-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - branch: development
+            tag: DEVELOPMENT-SNAPSHOT-2024-07-09-a
+
+            abi: arm64-v8a
+            options: -sdk ${env:SDKROOT}\..\..\..\Android.platform\Developer\SDKs\Android.sdk -sysroot ${env:ANDROID_NDK_ROOT}\toolchains\llvm\prebuilt\windows-x86_64\sysroot -I${env:SDKROOT}\..\..\..\..\..\..\Toolchains\0.0.0+Asserts\usr\include -Xlinker -zdefs
```

Comment:
Holy `\..\..\..\..\..\..\` batman!

Reply:
Hahaha 🦇 We need to dig through the toolchain layout since we don't have variables and dispatch is technically an unsupported component :(.